pkg/registry/core/pod/strategy.go

```diff
@@ -66,25 +65,21 @@ func (podStrategy) NamespaceScoped() bool {
 	return true
 }
 
-// PrepareForCreate clears fields that are not allowed to be set by end users on creation.
+// PrepareForCreate sets the default values and clears fields that are not
+// allowed to be set by end users on creation.
 func (podStrategy) PrepareForCreate(ctx context.Context, obj runtime.Object) {
 	pod := obj.(*api.Pod)
-	pod.Status = api.PodStatus{
-		Phase:    api.PodPending,
-		QOSClass: qos.GetPodQOS(pod),
-	}
-
+	podutil.SetDefaultPodValues(pod, nil)
```

Comment:
We should leave a nice comment explaining why we do this here instead defaults.go